lua/telekasten.lua

```diff
@@ -282,6 +282,14 @@ local function calculate_dates(date)
         .. ", "
         .. dinfo.year
 
+    local zonehour = string.sub(os.date("%z"), 1, 3)
+    local zonemin = string.sub(os.date("%z"), 4, 5)
+    dates.rfc3339 = os.date("%Y-%m-%dT%H:%M:%S")
```

Comment:
these don't take `dinfo` into account. So the `rfc3339` date will only work for the current time.

The function, however, is also used when selecting a date from the calendar. Please take this into account.